### PR TITLE
PORTF-1020 Add both basic Siklu support and btmp

### PR DESCRIPTION
**Pull request by yba-siklu**
## Description
This commits adds both basic Siklu dropbear support and also btmp
support, to the "new BSP" in three separate patches and a change to dropbear.mk.

## Tests
 * We do not have a test spec for dropbear, see note on testing on Jira issue PORTF-1020. Only basic monkey tests (ssh login to admin with correct/incorrect passwords) done.

## References
JIRA: PORTF-1020

### CODE REVIEW

**Review by dimaweber — APPROVED** [review]
Just to make sure: this is just part of PORTF-1020. btmp support for login will be added also. Right ?

**Comment by yba-siklu** [comment]
> Just to make sure: this is just part of [PORTF-1020](https://jira-siklu.atlassian.net/browse/PORTF-1020). btmp support for login will be added also. Right ?

Question not clear.
This PR *is* the implementation of PORTF-1020.

**Comment by dima1308** [comment]
> Question not clear.
> This PR is the implementation of PORTF-1020.

From what I see, this PR includes the implementation of PORTF-1020 and also additional patches from the original master. Am I missing something?

I see this PR is a draft. Is it ready to be merged?

**Comment by yba-siklu** [comment]
> > Question not clear.
> > This PR is the implementation of PORTF-1020.
> 
> From what I see, this PR includes the implementation of [PORTF-1020](https://jira-siklu.atlassian.net/browse/PORTF-1020) and also additional patches from the original master. Am I missing something?
> 
> I see this PR is a draft. Is it ready to be merged?

I have re-factored the patches from branch "master" according to the newer version of the C code in the dropbear package.
The only thing left before merge is for me to do the engineer test.

**Comment by dima1308** [comment]
> I have re-factored the patches from branch "master" according to the newer version of the C code in the dropbear package.
> The only thing left before merge is for me to do the engineer test.

The PR is approved, so when finish tests, we can merge.

**Pull request merged**
